### PR TITLE
add e2e testing support for ollinet in CI job

### PR DESCRIPTION
**Pull request by toliaqat**

### CODE REVIEW

File: tests/e2e/docker-compose.yml [hunk 1]
```diff
@@ -105,7 +107,7 @@ services:
     profiles:
       - synpress
     container_name: agoric_chain
-    image: ghcr.io/agoric/agoric-3-proposals:main
+    image: ghcr.io/agoric/agoric-3-proposals:${A3P_BRANCH}
```
Comment (turadg) [inline tests/e2e/docker-compose.yml:110]: this is a Docker tag, not branch.

what's publishing an 'ollinet' tag? I don't see one in https://github.com/Agoric/agoric-3-proposals/pkgs/container/agoric-3-proposals

**Review by turadg — CHANGES_REQUESTED** [review]

File: tests/e2e/docker-compose.yml (hunk 1 above)
Comment (toliaqat) [reply tests/e2e/docker-compose.yml:112]: this docker tag can only be used for local testing and not for ollinet.
for specific a3p tag on ollinet, I will have to make change in instagoric that I can do separately. 
removing this change for now.

Comment (toliaqat) [reply tests/e2e/docker-compose.yml]: @turadg updated the PR.

**Review by turadg — APPROVED** [review]

**Comment by rabi-siddique** [comment]
Fraz and I made the necessary changes to enable running E2E testing with other testnets besides `emerynet`. To prevent accidental merges, I am closing this pull request.

**Pull request closed without merging**